pkg/api/types.go

```diff
@@ -1362,6 +1362,8 @@ type PodSecurityContext struct {
 	// Use the host's ipc namespace.
 	// Optional: Default to false.
 	HostIPC bool `json:"hostIPC,omitempty"`
+	// The SeccompProfile is the seccomp profile this pod should run under.
```

Comment:
I'd appreciate a comment about the semantics wrt the per-container value and the per-pod value.  Is it a total override?  Additive?  Subtractive?